news/2024/2024-08-14-osu!-deskmats.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: post
+title: osu! deskmats
+date: 2024-08-14 15:00:00 +0000
+---
+
+You sit at your desk every day. You play osu! every day. You can see where we're going with this.
+
+![](/wiki/shared/news/2024-08-14-osu-deskmats/banner.jpg)
+
+A couple years ago, we celebrated osu!'s 15th anniversary with limited edition deskmats, and some of you pointed out that these "deskmats" were barely big enough to fit a keyboard.
```

Comment:
just searching around, "desk mat" with a space seems to be a lot more common (and has more search results)

Reply:
undecided here (i like "deskmat" more but that's not very convincing lol)

Reply:
going with deskmats as a \~stylistic choice\~